host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1.cmake

```diff
@@ -4,7 +4,13 @@
+
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH "ON" CACHE STRING "")
```

Comment:
yes its very large.... very very large. Maybe we can push back on Spack and get them to substitute the install prefix by default?

Reply:
It's been in lido as well, since we've already been using this spack version

Reply:
It may be safe to remove the CMAKE_INSTALL_RPATH_USE_LINK_PATH, CMAKE_BUILD_RPATH, CMAKE_INSTALL_RPATH sections. I tested it locally for quartz-clang and it passes tests and installs fine.

What do you mean by substitute the install prefix?